dts/arm/st/h7/stm32h743.dtsi

```diff
@@ -28,6 +28,10 @@
 			dma-requests= <107>;
 		};
 
+		dmamux2: dmamux@58025800 {
+			dma-requests= <107>;
+		};
```

Comment:
Would be nice to define for other H7 variants

Reply:
Added it for the other variants as well which defined `dmamux1`. Also found a mistake in `stm32h7a3` where `dma-requests` should be `107` and not `111`. Fixed it in a different commit. 

[Link to Reference Manual](https://www.st.com/resource/en/reference_manual/rm0455-stm32h7a37b3-and-stm32h7b0-value-line-advanced-armbased-32bit-mcus-stmicroelectronics.pdf)

![image](https://user-images.githubusercontent.com/39875036/221159238-10b45178-dcce-470f-b6a9-efad9f34ca04.png)